metadata_layer.py

```diff
@@ -313,22 +328,23 @@ def _display_name_to_sort_name(
         response = metadata_client.canonicalize_author_name(
             identifier_obj, self.display_name)
         sort_name = None
-        log = logging.getLogger("Abstract metadata layer")
-        if (response.status_code == 200
-            and response.headers['Content-Type'].startswith('text/plain')):
-            sort_name = response.content.decode("utf8")
-            log.info(
-                "Canonicalizer found sort name for %r: %s => %s",
-                identifier_obj,
-                self.display_name,
-                sort_name
-            )
+
+        if isinstance(response, basestring):
```

Comment:
In the MetadataWrangler, the response is a string direct from the Canonicalizer -- instead of a response object.